module-2/backend/week_one.md

```diff
@@ -5,30 +5,78 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+  GET - get an existing page
+  POST - create a new page
+  PUT - update an existing page
+  DELETE - delete a page
+  
 2. What is Sinatra?
-4. What is MVC?
-5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
-6. What types of variables are accessible in our view templates without explicitly passing them?
-7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
+  A Domain Specific language
+
+3. What is MVC?
+  MVC stands for Models, Views, and Controller. It is a convention commonly used to organize folders when developing web applications

```

Comment:
We have been using it to organize our folders/code for sure, but I think it also helps us to organize our thinking about how we split out tasks related to serving web pages.